codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/rustlang/CargoDependency.kt

```diff
@@ -208,7 +208,7 @@ data class CargoDependency(
         val TokioStream: CargoDependency = CargoDependency("tokio-stream", CratesIo("0.1.7"))
         val Tower: CargoDependency = CargoDependency("tower", CratesIo("0.4"))
         val Tracing: CargoDependency = CargoDependency("tracing", CratesIo("0.1"))
-
+        val Serde = CargoDependency("serde", CratesIo("1.0"), features = setOf("derive"), scope = DependencyScope.Compile)
```

Comment:
`DependencyScope.Compile` is the default so you don't need to include it when setting `features` with a named argument.
```suggestion
        val Serde = CargoDependency("serde", CratesIo("1.0"), features = setOf("derive"))
```

Reply:
Thank you!